### PR TITLE
Fix #165624 check-host.net

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

#165624

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -16962,8 +16962,8 @@ outlook.live.com#?#.customScrollBar[data-is-scrollable] > div[class] > div[class
 ! check-host.net
 check-host.net##div[id*="r"][id*="d"][id*="c"] img
 check-host.net##.text-left a > img[loading="lazy"]
-check-host.net#?#a[href]:contains(/protection|сайт/)
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^ ?[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ] ?$/):upward(1)
+check-host.net##a[href*="prometon.net"]
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^.*[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ].*$/):upward(1)
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:16966]: The rule is too wide because of using `.*`. I will change it a bit.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:16966]: Your rule breaks the site.
"https://check-host.net/ip-info?host=https://1.1.1.1"

Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:16966]: I tried doing the same thing but saw it broke the site, which is why I went with `.*`

Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:16966]: My rule did not break anything from what I could tell.

Comment (zloyden) [reply BaseFilter/sections/specific.txt:16966]: Well, don't worry - I've changed the rule. Thanks for letting me know about this.

**Pull request merged**